tests/test_ir/test_utils.py

```diff
@@ -22,6 +24,32 @@
 from clp_ffi_py.wildcard_query import WildcardQuery
 
 
+class JsonFileReader:
```

Comment:
Nit, it seems there is a specific name for Json files with each line as a Json object `JSON Lines`(.jsonl). https://jsonlines.org/
If you want, we can change the Readername to be JsonLineFileReader, and read_json_lines -> read_next_json_value?

Don't think it has to be in this change, but just wonder if we want to consider using `Json Lines`. It might cause some confusion for our customers though
@kirkrodrigues

Reply:
How about:

* `JsonFileReader` -> `JsonLinesFileReader`
* `read_json_lines` -> `read_line`
  * I don't think this should be plural since it yields after each line, right?

Reply:
it's yield from. so it actually yields a list of lines. I feel it should be plural

Reply:
1. Applied. I think the suggestions make sense.
2. It should not be plural as the generator returns object one by one. My bad.